vertical-pod-autoscaler/e2e/v1/e2e.go

```diff
@@ -236,7 +236,7 @@ func setupSuite() {
 	// #41007. To avoid those pods preventing the whole test runs (and just
 	// wasting the whole run), we allow for some not-ready pods (with the
 	// number equal to the number of allowed not-ready nodes).
-	if err := e2epod.WaitForPodsRunningReady(context.TODO(), c, metav1.NamespaceSystem, int32(framework.TestContext.MinStartupPods), int32(framework.TestContext.AllowedNotReadyNodes), podStartupTimeout); err != nil {
```

Comment:
Looks like we are losing the `framework.TestContext.AllowedNotReadyNodes` here, I haven't looked yet but do we know why? Is it OK?

Reply:
Found the original PR that removed this: https://github.com/kubernetes/kubernetes/pull/124205

So looks generally harmless? Would love to validate that `framework.TestContext.AllowedNotReadyNodes` was always 0 in our case as well.

Reply:
For what it's worth, I've run the e2e tests locally and everything passes